duckchat/duckchat-impl/src/main/java/com/duckduckgo/duckchat/impl/ui/SearchInterstitialFragment.kt

```diff
@@ -142,8 +140,7 @@ class SearchInterstitialFragment : DuckDuckGoFragment(R.layout.fragment_search_i
             exitInterstitial()
         }
         onDuckChatSent = { query ->
-            val data = Intent().putExtra(SearchInterstitialActivity.QUERY, query)
-            requireActivity().setResult(Activity.RESULT_CANCELED, data)
+            requireActivity().setResult(Activity.RESULT_CANCELED)
```

Comment:
This one was reported as a bug in https://app.asana.com/1/137249556945/project/1210424118459703/task/1210513464852533 (3)